torchgeo/datasets/mapinwild.py

```diff
@@ -0,0 +1,479 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""MapInWild dataset."""
+
+import os
+import shutil
+from collections.abc import Sequence
+from typing import Callable, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import rasterio
+import torch
+from einops import rearrange
+from torch import Tensor
+
+from torchgeo.datasets.geo import NonGeoDataset
+from torchgeo.datasets.utils import (
+    check_integrity,
+    download_url,
+    extract_archive,
+    percentile_normalization,
+)
+
+
+class MapInWild(NonGeoDataset):
+    """MapInWild dataset.
+
+    The `MapInWild <https://arxiv.org/abs/2212.02265>`_ dataset is curated for the task
+    of wilderness mapping on a pixel-level. MapInWild is a multi-modal dataset and
+    comprises various geodata acquired and formed from different RS sensors
+    over 1018 locations: dual-pol Sentinel-1, four-season Sentinel-2 with 10 bands,
+    ESA WorldCover map, and Visible Infrared Imaging Radiometer Suite
+    NightTime Day/Night band. The dataset consists of 8144 images with the shape
+    of 1920 × 1920 pixels. The images are weakly annotated from the
+    World Database of Protected Areas (WDPA).
+
+    Dataset features:
```

Comment:
Need empty line before start of list, otherwise it doesn't actually create a list: https://torchgeo--1131.org.readthedocs.build/en/1131/api/datasets.html#mapinwild